### PR TITLE
fix(image-cropper): update image cropper on sign up page

### PR DESCRIPTION
**Pull request by genie-magic**
**What this PR does** 📖
update image cropper on sign up page

**Which issue(s) this PR fixes** 🔨

AP-654

### CODE REVIEW

File: components/interactables/ImageCropper/ImageCropper.vue [hunk 1]
```diff
@@ -9,7 +9,8 @@ Vue.use(VueCroppie)
 
 const options = {
   type: 'base64',
-  size: { width: 600, height: 450 },
+  circle: false,
+  size: { width: 600, height: 600 },
   format: 'jpeg',
 }
```
Comment (molimauro) [inline components/interactables/ImageCropper/ImageCropper.vue:16]: Can we move these options in the `config.ts` file? :)

Comment (genie-magic) [reply components/interactables/ImageCropper/ImageCropper.vue]: @molimauro  I moved that. plz check

**Review by WanderingHogan — APPROVED** [review]
Great job, Eric

**Comment by molimauro** [comment]
Ty Eric!! :)

**Review by molimauro — APPROVED** [review]

**Review by stavares843 — APPROVED** [review]

**Pull request merged**